bin/ycsb.sh

```diff
@@ -0,0 +1,183 @@
+#!/bin/sh
```

Comment:
really? oh that's great. jeez I love shellcheck. Okay, POSIX it is.

Let me know if you'd like help getting the feedback to work.